### PR TITLE
Allow toggle between activity 'counts' and 'minutes' & period on Home…

### LINKED ISSUE #319: Allow toggle between activity 'counts' and 'minutes' on Home Profile, Activities section
The Home Profile, Activities section currently shows activity *counts* for the past 30 days, in several ways:

- by resident and type
- by activity type (all activities)
- by facilitator role (all activities)

screenshot from 2019-01-27 11-58-54

# Task
Allow users to toggle between activity *counts* and *minutes*, with a server method to prepare the data.

screenshot from 2019-01-27 11-58-14

BONUS: allow the user to toggle between 7 and 30 day timespan:

screenshot from 2019-01-27 11-54-07

## Related issues
The following issues are related, and might be good to be done in conjunction with this one:
- #317 
- #318

### PR DESCRIPTION
**Pull request by marla-singer**
… Profile, Activities section #319 
Closes #319

### CODE REVIEW

File: app/client/views/home/activities/counts/activity_type/activity_type.js [hunk 1]
```diff
@@ -2,16 +2,26 @@ Template.homeActivityCountsByActivityType.onCreated(function () {
...
 
-  // Call method to fetch data, assigning it to reactive variable
-  Meteor.call('getHomeActivityTypeCountsLast30days', homeId, function (error, chartData) {
-    // update chart data reactive variable
-    templateInstance.chartData.set(chartData);
+  this.autorun(() => {
+    const activityPeriod = Template.currentData().activityPeriod;
+
+    // Not fetch data if period isn't changed
+    if (currentActivityPeriod !== activityPeriod) {
```
Comment (brylie) [inline app/client/views/home/activities/counts/activity_type/activity_type.js]: This should be unnecessary, as autorun will only re-run when the reactive data source changes. Is this re-running when the value doesn't change?

File: app/client/views/home/home.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ Template.home.onCreated(function () {
   // Set current Home ID from router
   templateInstance.homeId = Router.current().params.homeId;
 
+  // Create reactive variables to hold activity metric and period
+  templateInstance.activityMetric = new ReactiveVar('count');
```
[the remark this answers is not in the document]
Comment (marla-singer) [reply app/client/views/home/home.js]: I did it before. But it doesn't work and values are undefined. When we are getting values html isn't ready.

**Comment by marla-singer** [comment]
@brylie Done. All request changes are fixed

File: app/client/views/home/home.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (brylie) [reply app/client/views/home/home.js]: Did you try it in `onCreated` or `onRendered`?

[the remark this answers is not in the document]
Comment (marla-singer) [reply app/client/views/home/home.js]: I tried `onRendered`
Yes, it should be. But data is fetching in that moment. So it's displaying spinner at all screen and no toggle buttons.

**Comment by marla-singer** [comment]
@brylie If everything is fine let's merge it :)

**Pull request merged**